exchanges/btcmarkets/btcmarkets.go

```diff
@@ -379,52 +553,55 @@ func (b *BTCMarkets) SendHTTPRequest(path string, result interface{}) error {
 }
 
 // SendAuthenticatedRequest sends an authenticated HTTP request
-func (b *BTCMarkets) SendAuthenticatedRequest(reqType, path string, data, result interface{}) (err error) {
+func (b *BTCMarkets) SendAuthenticatedRequest(method, path string, data map[string]interface{}, result interface{}, specialCase []interface{}) (err error) {
```

Comment:
you can probably change data to an interface{} and using the switch case test for its type map[string]interface{} or []interface{} and get rid of special case param